samples/bluetooth/central_and_peripheral_hr/README.rst

```diff
@@ -33,24 +33,22 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf21540dk_nrf52840
 
-Bluetooth® LE Central Role only:
+For testing just the Bluetooth® LE Central Role operation the sample requires one of the following setups:
 
-  The sample requires a smartphone or a tablet running a compatible application.
+  * A smartphone or a tablet running a compatible application.
 
-  You can test the application with the :ref:`zephyr:peripheral_hr` sample.
+  * Another development kit running the :ref:`zephyr:peripheral_hr` sample.
   See the documentation for that sample for detailed instructions.
 
-Relay mode:
+To test the Relay mode operation you need one of the following setups:
```

Comment:
```suggestion
To test the Relay mode operation, you need the following setup:
```

Reply:
Fixed in a bit different way to emphasize the bullet list is an alternative.

Reply:
Same with this, but have the comma there after "operation" ;)

Reply:
done